docs/topics/testing/tools.txt

```diff
@@ -1511,9 +1517,9 @@ your test suite.
 
 .. method:: SimpleTestCase.assertContains(response, text, count=None, status_code=200, msg_prefix='', html=False)
 
-    Asserts that a ``Response`` instance produced the given ``status_code`` and
-    that ``text`` appears in the content of the response. If ``count`` is
-    provided, ``text`` must occur exactly ``count`` times in the response.
+    Asserts that a :class:`response <django.http.HttpResponse>` produced the
+    given ``status_code`` and that ``text`` appears its content. If ``count``
```

Comment:
`appears its content` => `appears in the content`?
Could also be an opportunity to link to the `.content` attribute, maybe? ... Although perhaps not if it's not already linking to `.status_code`

Reply:
Oops, I missed the `in`, well spotted thanks.

I've also pushed a commit with added links to `HttpResponse.status_code` and `HttpResponse.content`.
It's hard to tell with the local sphinx theme how it will look once deployed, but I'm not sure if it makes things clearer or not.
There's doesn't seem to be any visual difference between links like `response` or `status_code` and plain formatted words like `text` or `count`:
![Screenshot 2021-11-22 at 19-01-46 Testing tools — Django 4 1 dev20211122175334 documentation](https://user-images.githubusercontent.com/6345/142912598-58f79ba4-6c9a-491c-a508-117bdcbd93e2.png)